src/components/course-review-card.js

```diff
@@ -5,11 +5,17 @@ import { Card } from 'semantic-ui-react';
 import { Rating } from 'semantic-ui-react';
 
 const CourseReviewCard = (props) => {
-  const { code, name, desc } = props;
+  const { code, name } = props;
+  let { desc } = props;
+
+  // Truncate desc and add elipsis to end if too long
+  if (desc.length > 220) {
```

Comment:
we always want to use `const`

what you can do here is

```
const { code, name, desc } = props;

const displayDesc = desc.length > 220 ? desc.substring(220, length) + '...' : desc;
```

and then use `displayDesc` from then on

Reply:
also where did you declare `length`? soz if blind